app/src/features/FhirAttributePanel/AttributeInputGroup.tsx

```diff
@@ -157,45 +157,84 @@ const AttributeInputGroup = ({
         id: inputGroup.id,
       }).unwrap();
     } catch (e) {
-      //
+      enqueueSnackbar(e.error, { variant: "error" });
+    }
+  };
+
+  const handleCreateCondition = useCallback(() => {
+    const newCondition: Partial<ConditionType> = {
+      input_group: inputGroup.id,
+      action: "INCLUDE",
+    };
+    setConditions([...conditions, newCondition]);
+  }, [conditions, inputGroup.id]);
+
+  // Auto-create a condition when none exists for InputGroups with index > 0
+  useEffect(() => {
+    const isInputGroupConditionsEmpty =
+      apiConditions?.length === 0 && conditions.length === 0;
+    if (
+      hasApiConditionsLoaded &&
+      isInputGroupConditionsEmpty &&
+      inputGroupIndex > 0
+    ) {
+      handleCreateCondition();
+    }
+  }, [
+    apiConditions?.length,
+    conditions.length,
+    handleCreateCondition,
+    hasApiConditionsLoaded,
+    inputGroupIndex,
+  ]);
+
+  const handleMenuClick = (e: React.MouseEvent<HTMLButtonElement>) =>
+    popupState.open(e);
+
+  const handleCreateSqlInput = async () => {
+    if (mapping) {
+      try {
+        const inputColumn = await createColumn({
+          columnRequest: {
+            table: mapping.primary_key_table,
+            column: mapping.primary_key_column,
+            owner: mapping.primary_key_owner,
```

Comment:
explain why we send the primary key table/column here

Reply:
For UX purpose as the input column will already be filled with those values